FogCreek/Glitch-Community
# Profuse tea: Fix ssr project pages

## Links
https://profuse-tea.glitch.me/
https://app.clubhouse.io/glitch/story/3079/window-is-not-defined-when-trying-to-ssr-project-pages

## Changes:
- Replace `useWindowDimensions` with `useWindowSize`, which waits until after the first render to check window
- Project actions use the wide style if `width` is not ready yet

## How To Test:
Load up a project page (one you own and one you do not) and make sure it server renders and responds to resizing the window

## Feedback I'm looking for:
Pages have gotten broken in ssr a few times, because it's easy not to notice if you don't have the server logs open. Should the site do more to advertise when ssr fails? (for ex logging something in the browser console or failing the request entirely)
